schema.sql

```diff
@@ -42,4 +42,29 @@ ALTER TABLE animals
   ADD owner_id,
   ADD CONSTRAINT fk_owners
   FOREIGN KEY (owner_id) 
-  REFERENCES owners (id);
\ No newline at end of file
+  REFERENCES owners (id);
+
+  -- Create vets table
+  CREATE TABLE vets (
+      id BIGSERIAL PRIMARY KEY NOT NULL,
+      name VARCHAR(100),
+      age INT NOT NULL,
+      date_of_graduation DATE NOT NULL
+  );
+
+  --Specialization table
+CREATE TABLE specializations (
+    species_id INT NOT NULL,
+    vet_id INT NOT NULL,
+    FOREIGN KEY (species_id) REFERENCES species (id),
+    FOREIGN KEY (vet_id) REFERENCES vets (id) 
+);
+
+--Visit table
+CREATE TABLE visits (
+    animal_id INT NOT NULL,
```

Comment:
![image](https://user-images.githubusercontent.com/65068771/161147261-e960b783-1014-4363-b4ef-dbe6002f1d8b.png)

- [ ] Kindly put the `;` at the end of the code in the screenshot :point_up:  above because is throwing an error when rung your `schema.sql` file. 

this is the error :point_down: 

![image](https://user-images.githubusercontent.com/65068771/161147462-e64acc25-2a5c-488d-8752-327c1522173e.png)